HW8/Game/src/Main.java

```diff
@@ -2,37 +2,27 @@ public class Main {
     public static void main(String[] args) {
 
 
-        Character First = new Character();
-        Character Second = new Character();
+        Character firstCharacter = new Character("First", 100, 1, 3);
+        Character secondCharacter = new Character("Second", 150, 2, 0);
 
-        First.name = "First";
-        First.defence = 3;
-        First.strength = 1;
-        First.health = 100;
 
-        Second.name = "Second";
-        Second.defence = 0;
-        Second.strength = 2;
-        Second.health = 150;
+        System.out.println(Character.characterCount);
 
+        while (firstCharacter.isAlive() && secondCharacter.isAlive()){
+            firstCharacter.attack(secondCharacter);
+            firstCharacter.displayStatus();
+            secondCharacter.displayStatus();
 
-        System.out.println(Character.CHARACTER_COUNT);
-
-        while (First.isAlive() && Second.isAlive()){
-            First.attack(Second);
-            First.displayStatus();
-            Second.displayStatus();
-
-            Second.attack(First);
-            First.displayStatus();
-            Second.displayStatus();
+            secondCharacter.attack(secondCharacter);
+            firstCharacter.displayStatus();
+            secondCharacter.displayStatus();
         }
 
```

Comment:
According to the task we wanted to display the winner after the fight was over :)